app/src/generate.py

```diff
@@ -66,5 +72,13 @@ def generate(
             },
             {"content": query, "role": "user"},
         ]
-    response = completion(model="gpt-4o", messages=messages)
+
+    logger.info("Calling %s for query: %s", llm, query)
+    response = completion(model=llm, messages=messages, **completion_args(llm))
     return response["choices"][0]["message"]["content"]
+
+
+def completion_args(llm: str) -> dict[str, Any]:
+    if llm.startswith("ollama/"):
+        return {"api_base": os.environ["OLLAMA_HOST"]}
```

Comment:
Apparently, setting `OLLAMA_HOST` isn't sufficient. LiteLLM continues to use the default `localhost` unless `api_base` is passed in as an argument.